### PR TITLE
Add upcoming events to slider.

### PR DESCRIPTION
**Pull request by tpmccallum**

### CODE REVIEW

File: content/events/2023-civo-navigate.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+title = "Civo Navigate Europe 2023"
+date = "2023-08-21T00:00:01Z"
+description = "Exciting times await you at the Civo Navigate Europe 2023 event this September! Find us at our vibrant booth. We also have a line-up of engaging sessions waiting for you: a workshop, a breakout session, and your chance to explore the dynamic world of WebAssembly with us. Bring your passion, your questions, and your energy. And remember, amidst all the learning, there’s also room for some fun!"
+
+[extra]
+type = "event"
+eventdates = "September 5th & 6th - London, UK"
```
Comment (itowlson) [inline content/events/2023-civo-navigate.md:7]: Huge nitpick but for the AI one we used the `26th-27th` format rather than `26th & 27th`

**Comment by karenhchu** [comment]
Are we standardizing adding the city to the same line as the event date?

**Review by itowlson — APPROVED** [review]

File: content/events/2023-civo-navigate.md (hunk 1 above)
Comment (karenhchu) [reply content/events/2023-civo-navigate.md:7]: I think we should use "-" bc some events will span more than 2 days.

File: content/events/2023-civo-navigate.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+title = "Civo Navigate Europe 2023"
+date = "2023-08-21T00:00:01Z"
```
Comment (flynnduism) [inline content/events/2023-civo-navigate.md:2]: Civo Navigate [takes place](https://www.civo.com/navigate) Sept 5th and 6th

File: content/events/2023-civo-navigate.md (hunk 1 above)
Comment (tpmccallum) [inline content/events/2023-civo-navigate.md:7]:
```suggestion
eventdates = "September 5th-6th - London, UK"
```

File: content/events/2023-ai-conference.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+description = "Join us along with AI experts and researchers. Whether you're a student, professional, entrepreneur, or just an enthusiast, we would love to meet you! Drop by our booth so we can get to know each other, learn about what we are working on, and you can grab some swag. AI is an amazing emerging field and this could be a monumental step in your professional development. See you there!"
+
+[extra]
+type = "event"
+eventdates = "September 26th-27th - San Francisco, California"
+eventexpires = "2023-09-28T01:01:01Z"
+url = "https://aiconference.com/"
+image = "static/image/avatar.png"
+
+---
```
Comment (tpmccallum) [inline content/events/2023-ai-conference.md:12]:
```suggestion
---

```

File: content/events/2023-civo-navigate.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+description = "Exciting times await you at the Civo Navigate Europe 2023 event this September! Find us at our vibrant booth. We also have a line-up of engaging sessions waiting for you: a workshop, a breakout session, and your chance to explore the dynamic world of WebAssembly with us. Bring your passion, your questions, and your energy. And remember, amidst all the learning, there’s also room for some fun!"
+
+[extra]
+type = "event"
+eventdates = "September 5th-6th - London, UK"
+eventexpires = "2023-09-07T01:01:01Z"
+url = "https://www.civo.com/navigate"
+image = "static/image/avatar.png"
+
+---
```
Comment (tpmccallum) [inline content/events/2023-civo-navigate.md:12]:
```suggestion
---

```

**Review by tpmccallum — COMMENTED** [review]
Adding an extra line after the `---` fixes the errors we have been having @karenhchu 
I will keep this in mind for next time; I could not for the life of me figure this out. But eventually did :)

File: content/events/2023-civo-navigate.md (hunk 1 above)
Comment (tpmccallum) [reply content/events/2023-civo-navigate.md]: Thanks @karenhchu and I will also abbreviate the month, as you have done (so the new events all match).

File: content/events/2023-ai-conference.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+title = "The AI Conference 2023"
+date = "2023-09-05T17:00:00Z"
+description = "Join us along with AI experts and researchers. Whether you're a student, professional, entrepreneur, or just an enthusiast, we would love to meet you! Drop by our booth so we can get to know each other, learn about what we are working on, and you can grab some swag. AI is an amazing emerging field and this could be a monumental step in your professional development. See you there!"
+
+[extra]
+type = "event"
+eventdates = "September 26th-27th - San Francisco, California"
```
Comment (tpmccallum) [inline content/events/2023-ai-conference.md:7]:
```suggestion
eventdates = "Sept 26-27 - San Francisco, California"
```

File: content/events/2023-civo-navigate.md [hunk 6]
```diff
@@ -0,0 +1,12 @@
+title = "Civo Navigate Europe 2023"
+date = "2023-08-21T00:00:01Z"
+description = "Exciting times await you at the Civo Navigate Europe 2023 event this September! Find us at our vibrant booth. We also have a line-up of engaging sessions waiting for you: a workshop, a breakout session, and your chance to explore the dynamic world of WebAssembly with us. Bring your passion, your questions, and your energy. And remember, amidst all the learning, there’s also room for some fun!"
+
+[extra]
+type = "event"
+eventdates = "September 5th-6th - London, UK"
```
Comment (tpmccallum) [inline content/events/2023-civo-navigate.md:7]:
```suggestion
eventdates = "Sept 5-6 - London, UK"
```

**Pull request merged**